### PR TITLE
Add an example showing how to compare traces over time with Splunk Enterprise

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

**Review by tmorrisonsfx — CHANGES_REQUESTED** [review]
Changes requested.

File: examples/splunk-hec-traces/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# HEC Traces Example
+
+This example showcases how the agent works with Splunk Enterprise and traces.
+
+The example runs as a Docker Compose deployment. The collector can be configured to send traces to Splunk Enterprise.
+
+Splunk is configured to receive data from the OpenTelemetry Collector using the HTTP Event collector. To learn more about HEC, visit [our guide](https://dev.splunk.com/enterprise/docs/dataapps/httpeventcollector/).
```
[the remark this answers is not in the document]
Comment (atoulme) [reply examples/splunk-hec-traces/README.md]: Will use Splunk Enterprise throughout. Fixed sentence. Using Splunk OpenTelemetry Connector throughout.

File: examples/splunk-hec-traces/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+```bash
+$> docker-compose up --build
+```
+You can stop the example by pressing Ctrl+C.
+
+Splunk will become available on port 18000. You can login on [http://localhost:18000](http://localhost:18000) with `admin` and `changeme`.
+
+Once logged in, visit the [spans comparisons dashboard](http://localhost:18000/en-US/app/search/spans_time_comparisons) to see a comparison of traces over time sent by the OpenTelemetry Collector.
+
+You can also search traces for information, such as querying by tag or any element of traces, by searching the traces index `index=traces`
```
[the remark this answers is not in the document]
Comment (atoulme) [reply examples/splunk-hec-traces/README.md]: Added period to the sentence.

**Comment by atoulme** [comment]
Thanks for the review @tmorrisonsfx ! Please review my changes at leisure.

File: examples/splunk-hec-traces/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+This example showcases how the agent works with Splunk Enterprise and traces.
+
+The example runs as a Docker Compose deployment. The collector can be configured to send traces to Splunk Enterprise.
+
+Splunk Enterprise is configured to receive data from the Splunk OpenTelemetry Connector using the HTTP Event Collector. To learn more about HEC, see [Configure the Splunk HTTP Event Collector](https://dev.splunk.com/enterprise/docs/dataapps/httpeventcollector/) for use with additional technologies.
+
+To deploy the example:
+1. Check out the [Splunk OpenTelemetry Connector repository](https://github.com/signalfx/splunk-otel-collector)
+2. Open a terminal
```
Comment (tmorrisonsfx) [inline examples/splunk-hec-traces/README.md:11]: Can you add punctuation to steps 1 and 2?

File: examples/splunk-hec-traces/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+3. Type: the following commands:
+```bash
+$> cd examples/splunk-hec-traces
+$> docker-compose up --build
+```
+You can stop the example by pressing Ctrl+C.
```
Comment (tmorrisonsfx) [inline examples/splunk-hec-traces/README.md:17]: Suggested change: `Ctrl + C`

**Review by tmorrisonsfx — APPROVED** [review]
A few minor suggestions. Thanks!

**Comment by atoulme** [comment]
Thanks, changes applied, ready for merge.

**Review by tmorrisonsfx — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Pull request merged**